draft-packages/illustration/KaizenDraft/Illustration/Players/VideoPlayer.tsx

```diff
@@ -120,69 +122,48 @@ export const VideoPlayer = ({
     // Add event listeners for the video element
     const { current: videoElement } = videoRef
     if (!videoElement || !onEnded) return
+
     videoElement.addEventListener("ended", onEnded)
 
     return function cleanup() {
       videoElement.removeEventListener("ended", onEnded)
     }
   }, [videoRef])
 
-  const aspectClassName =
-    (aspectRatio ? styles[aspectRatio] : "") + " " + styles.aspectRatioWrapper
-
-  return (
-    <>
-      {aspectRatio ? (
-        <figure className={aspectClassName}>
-          <video
-            muted={true}
-            aria-hidden={true}
-            preload="metadata"
-            ref={videoRef}
-            width="100%"
-            data-testid="kz-video-player"
-            className={styles.wrapper}
-            loop={loop}
-            poster={assetUrl(`${fallback}.png`)}
-            autoPlay={prefersReducedMotion ? false : autoplay}
-            playsInline={true}
-          >
-            {/**
-             * This seems counter-intuitive, but webm support is codec specific.
-             * Only offer webm if we are positive the browser supports it.
-             * Reference: https://bugs.webkit.org/show_bug.cgi?id=216652#c1
-             */}
-            {canPlayWebm() && (
-              <source src={assetUrl(`${source}.webm`)} type="video/webm" />
-            )}
-            <source src={assetUrl(`${source}.mp4`)} type="video/mp4" />
-          </video>
-        </figure>
-      ) : (
-        <video
-          muted={true}
-          aria-hidden={true}
-          preload="metadata"
-          ref={videoRef}
-          width="100%"
-          data-testid="kz-video-player"
-          className={styles.wrapper}
-          loop={loop}
-          poster={assetUrl(`${fallback}.png`)}
-          autoPlay={prefersReducedMotion ? false : autoplay}
-          playsInline={true}
-        >
-          {/**
-           * This seems counter-intuitive, but webm support is codec specific.
-           * Only offer webm if we are positive the browser supports it.
-           * Reference: https://bugs.webkit.org/show_bug.cgi?id=216652#c1
-           */}
-          {canPlayWebm() && (
-            <source src={assetUrl(`${source}.webm`)} type="video/webm" />
-          )}
-          <source src={assetUrl(`${source}.mp4`)} type="video/mp4" />
-        </video>
+  const videoPlayer = (
```

Comment:
```suggestion
  const VideoPlayer = (
```

Since it's a JSX Element, we should use PascalCase (might need to be renamed though of course, since it conflicts with the component name)

Reply:
I contemplated using PascalCase but decided against, because then it might look too much like a component and I'm not doing `...<VideoPlayer />...` but `...{videoPlayer}...`.

Reply:
Yeah it's a funny one. But it probably is moreso a subcomponent than a variable. I'd make it a component, but eh, not a big deal. Up to you whether you change it or not.